### PR TITLE
[PREVIEW] Introduce polling interval to speed up tests which wait for an event

### PR DESCRIPTION
**Pull request by luigibk**
### Change description ###
Introduce polling interval to speed up tests which wait for an event

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/BlobProcessorTaskTest.java [hunk 1]
```diff
@@ -200,6 +202,7 @@ public void should_keep_zip_file_after_unsuccessful_upload_and_not_create_doc_pr
         CloudBlockBlob blob = testContainer.getBlockBlobReference(zipFile);
         await("file should not be deleted")
             .timeout(2, SECONDS)
+            .pollInterval(1, TimeUnit.SECONDS)
```
Comment (rkondratowicz) [inline src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/BlobProcessorTaskTest.java]: Here we need to wait the full time (the background job may have not even checked the file)

File: src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/BlobProcessorTaskTest.java [hunk 2]
```diff
@@ -299,6 +303,7 @@ public void should_not_delete_blob_unless_processed()
         CloudBlockBlob blob = testContainer.getBlockBlobReference(VALID_ZIP_FILE_WITH_CASE_NUMBER);
         await("file should not be deleted")
             .atMost(5, SECONDS)
+            .pollInterval(1, TimeUnit.SECONDS)
```
Comment (rkondratowicz) [inline src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/BlobProcessorTaskTest.java]: same, here. Check file still exists after a while

File: src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/BlobProcessorTaskTest.java (hunk 1 above)
Comment (luigibk) [reply src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/BlobProcessorTaskTest.java]: In fact it waits until the timeout expires unless the file is deleted before that. This way it just the condition more often...

File: src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/BlobProcessorTaskTest.java (hunk 2 above)
Comment (luigibk) [reply src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/BlobProcessorTaskTest.java]: Same as above. It waits the full timeout unless there is no need for it.

File: src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/BlobProcessorTaskTest.java (hunk 1 above)
Comment (rkondratowicz) [reply src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/BlobProcessorTaskTest.java]: To clarify, here we check that something *didn't happen*, so we can't check immediately.
Other cases are fine.

**Pull request closed without merging**